src/doc/builtinplugins.rst

```diff
@@ -1731,6 +1727,13 @@ attributes are supported:
      - ptr
      - Pointer to a ``Filesystem::IOProxy`` that will handle the I/O, for
        example by reading from memory rather than the file system.
+   * - ``pnm:bigendian``
+     - int
+     - If nonzero, the PNM file is big-endian (the default is little-endian).  
+   * - ``pnm:pfmflip``
+      - int
+      - If nonzero, the PFM (float) file is flipped upside-down (the default
+      is flipped).
```

Comment:
This is a little confusing. Is the default to flip or not to flip?

Reply:
Flipped. Because of GIMP and NetPBM that have this by default flipped. As result libjxl tools also export flipped pfm.

When I personally prefer non flipped